src/client/src/rt-theme/colors.js

```diff
@@ -0,0 +1,203 @@
+import _ from 'lodash';
```

Comment:
No more loadash!, Also if you really need it just pull what you need

Reply:
That's build step `babel-plugin-lodash`. In answer, no.